### PR TITLE
Fix filesystem permissions after build

### PR DESCRIPTION
**Pull request by jgmize**
Prevent jenkins errors like the following: https://ci.us-west.moz.works/blue/rest/organizations/jenkins/pipelines/irlpodcast/branches/dev/runs/55/nodes/6/steps/7/log/?start=0

### CODE REVIEW

**Comment by jgmize** [comment]
This is now working locally, and should be safe to merge. However, this only addresses the symptom by fixing the filesystem permissions-- we should not be running anything as root, even in a container. Unfortunately the `huli/gulp` docker image did not work for me when I tried overriding the user with `docker -u`. We will may need to find (or build) an alternative docker image to run gulp in.

File: bin/build.sh [hunk 1]
```diff
@@ -5,8 +5,9 @@ mkdir -p ./release
 # let gulp build the assets
 docker run --rm --label=gulp \
     --volume=$(pwd):/srv \
+    --entrypoint /bin/bash \
     huli/gulp \
-    build --production
+    -c "gulp build --production && chown $(id -u):$(id -g) . -R"
```
Comment (pmac) [inline bin/build.sh]: I usually just run the container as the user with `--user "$(id -u)"`. Just for extra info.

Comment (jgmize) [reply bin/build.sh]: Yes, the `-u` flag I was referring to in https://github.com/mozilla/irlpodcast/pull/174#issuecomment-463844594 is the shorthand for `--user`, and I got an error due to issues with the way the `huli/gulp` image tries to write files to the root of the container.

Comment (pmac) [reply bin/build.sh]: Ah! I totally missed your other comment mentioning the `-u`. Makes sense. Thanks for the clarification.

Comment (jgmize) [reply bin/build.sh]: Specifically, the error I got was:
```
$ bin/build.sh
npm WARN IRLPodcast@2.0.0 license should be a valid SPDX license expression
npm ERR! Linux 4.9.125-linuxkit
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "--save-dev"
npm ERR! node v6.3.1
npm ERR! npm  v3.10.3
npm ERR! path /.npm
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall mkdir

npm ERR! Error: EACCES: permission denied, mkdir '/.npm'
npm ERR!     at Error (native)
npm ERR!  { Error: EACCES: permission denied, mkdir '/.npm'
npm ERR!     at Error (native) errno: -13, code: 'EACCES', syscall: 'mkdir', path: '/.npm' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.

npm ERR! Please include the following file with any support request:
npm ERR!     /srv/npm-debug.log
Error: Unable to open Config file.
 (open config-.toml: no such file or directory)
```

**Pull request merged**